analysis_options.yaml

```diff
@@ -61,7 +61,7 @@ linter:
   - prefer_typing_uninitialized_variables
   - recursive_getters
   - slash_for_doc_comments
-  - super_goes_last
+  #- super_goes_last
```

Comment:
Add a comment as to why we are commentin out this linter?

Reply:
Oh wait, obsolete linter. Can we just delete it?

Reply:
would prefer to delete it entirely.  will do so.